modules/imgcodecs/src/grfmt_webp.cpp

```diff
@@ -126,8 +84,39 @@ bool WebPDecoder::readHeader()
     WebPBitstreamFeatures features;
     if (VP8_STATUS_OK == WebPGetFeatures(header, sizeof(header), &features))
```

Comment:
Just to reduce the imbricated `if` , write it as:
```cpp
if (VP8_STATUS_OK == WebPGetFeatures(header, sizeof(header), &features)) return false;
```
And then you can remove the big `{}` below.